src/base/preferences.cpp

```diff
@@ -1403,6 +1403,16 @@ void Preferences::setOsCache(bool enable)
     setValue("Preferences/Advanced/osCache", enable);
 }
 
+bool Preferences::getGuidedReadCache() const
```

Comment:
This whole PR is outdated, I'm not even sure if @sledgehammer999 would approve this...